packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.stories.ts

```diff
@@ -36,19 +67,56 @@ const metadata: Meta<RichTextEditorArgs> = {
         componentName: 'rich text editor',
         statusLink: 'https://github.com/ni/nimble/issues/1288'
     })}
-    <${richTextEditorTag}>
+    <${richTextEditorTag}
+        ${ref('editorRef')}
+        data-unused="${x => x.setMarkdownData(x)}"
+    >
         ${when(x => x.footerActionButtons, html`
             <${buttonTag} appearance="ghost" slot="footer-actions">Cancel</${buttonTag}>
             <${buttonTag} slot="footer-actions">OK</${buttonTag}>`)}
     </${richTextEditorTag}>
     `),
     argTypes: {
+        data: {
+            name: 'setMarkdown(value)',
```

Comment:
I noticed that when I toggle from "Plain string" to "...all supported...", the editor ends up in a state where some of the toggle buttons are active but the cursor is not present inside the editor. This was unexpected to me: I thought the buttons would only be active to reflect the state of the text at the cursor. If the editor doesn't have focus, I'd expect all the buttons to be toggled off. Instead it's behaving as if the cursor is at the end of the text even thought the cursor isn't visible.

This is only a minor quirk so if it's a challenge to fix then I'm ok leaving it as-is, but I think it's worth spending some time investigating it.

<img width="372" alt="image" src="https://github.com/ni/nimble/assets/10500124/eb23a850-38a5-423a-93cf-0f342e1e9f39">

Reply:
This is the expected behavior in [ TipTap editor ](https://tiptap.dev/examples/default), that it will make the toggle button active based on the end of the content being set.

One way is to add editor focus programmatically when setting markdown, which will make the cursor active at the document

```
   public setMarkdown(markdown: string): void {
        const html = this.getHtmlContent(markdown);
        this.tiptapEditor.commands.setContent(html);
        this.tiptapEditor.commands.focus()
  }
```

Reply:
Hmm, I'd be wary of having the `setMarkdown()` method change the focus. It's a side effect that would be unexpected to me and the multiple editor case you mentioned would also be problematic.

If this is TipTap's default behavior then let's leave it as-is and see if any users have feedback about it.